packages/redux/src/checkout/actions/factories/createCheckoutOrderContextFactory.ts

```diff
@@ -0,0 +1,63 @@
+import * as actionTypes from '../../actionTypes.js';
+import {
+  type CheckoutOrder,
+  type Config,
+  type PostCheckoutOrderContext,
+  type PostCheckoutOrderContextData,
+  type PostCheckoutOrderContextResponse,
+  toBlackoutError,
+} from '@farfetch/blackout-client';
+import type { CreateCheckoutOrderContextAction } from '../../types/index.js';
+import type { Dispatch } from 'redux';
+
+/**
+ * Method responsible for creating a checkout order context.
+ *
+ * @param postCheckoutOrderContext - Post checkout order context client.
+ *
+ * @returns Thunk factory.
+ */
+const createCheckoutOrderContext =
+  (postCheckoutOrderContext: PostCheckoutOrderContext) =>
+  (
+    checkoutOrderId: CheckoutOrder['id'],
+    data: PostCheckoutOrderContextData,
+    config?: Config,
+  ) =>
+  async (
+    dispatch: Dispatch<CreateCheckoutOrderContextAction>,
+  ): Promise<PostCheckoutOrderContextResponse> => {
+    try {
+      dispatch({
+        type: actionTypes.CREATE_CHECKOUT_ORDER_CONTEXT_REQUEST,
```

Comment:
The actions that are being dispatched by this thunk are not being handled anywhere so the called cannot know the state of the request. Is this intended?

Reply:
I added 2 handlers to control the create and remove, can you check the `reducer.ts` file again?